sBTC SIP

```diff
@@ -0,0 +1,166 @@
+# sBTC SIP
+
+## SIP Number: 022
+**Title:** A Decentralized Two-Way Bitcoin Peg  
+**Type:** Standard  
+**Status:** Draft  
+
+## Abstract
+sBTC is a novel digital asset that lets you move Bitcoin in and out of the Stacks blockchain. With sBTC, users can interact with Clarity smart contracts, which enable Bitcoin applications such as payments, decentralized lending, decentralized exchanges, and BTC-backed stablecoins.
```

Comment:
Recommend dropping this paragraph.  It's misleading in places (e.g. Bitcoin does not "move" into Stacks or out of Stacks), and doesn't tell us anything that the subsequent paragraphs already tell us ("users can interact with Clarity contracts, which enable Bitcoin applications ... " as opposed to what?)

Reply:
This paragraph is slightly redundant - removed.